src/pip/_internal/compat.py

```diff
@@ -192,3 +193,38 @@ def samefile(file1, file2):
         path1 = os.path.normcase(os.path.abspath(file1))
         path2 = os.path.normcase(os.path.abspath(file2))
         return path1 == path2
+
+
+if sys.version_info >= (3, 3):
+    """Returns a tuple (x, y) representing the width(x) and the height(y)
+    in characters of the terminal window."""
+    def get_terminal_size():
+        cr = shutil.get_terminal_size()
+        return int(cr[1]), int(cr[0])
```

Comment:
Isn't the return value a tuple of integers? There's no need to call `int` here.

Reply:
I have a doubt. `shutil.get_terminal_size` returns a tuple of type `os.get_terminal_size`. if I change the definition for Python >= 3.3 to just `retrun shutil.get_terminal_size()` would that be fine? Can the fact that the returned tuple is of another type could cause a problem somewhere else?

Reply:
I just went and looked up the docs. We don't even need to flip the order like this function is doing right now.

So, yes, this can just be:

```py
def get_terminal_size():
    return tuple(shutil.get_terminal_size())
```

> Can the fact that the returned tuple is of another type could cause a problem somewhere else?

I don't think so but I'd not bother with that and let it stay a tuple in this PR. If you want, you can follow up with a PR that changes this to a namedtuple and at the call-sites, do attribute access for clarity.